producers/cube_config_mfx.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+
+# Custom bins
+def binBoundaries(run):
+    if isinstance(run,str):
+        run=int(run)
+    if run>0:
+        # return np.arange(-5.,50.,0.2)
+        return np.arange(-5.,10.,1)
+    return None
+
+
+# ##### FILTERS ##########################################################################
+# filters to apply to the data
+# format: list of [det (field), low, high, name]
+# 'filter1' is the standard name and will not be added to the h5 file name.
+filters = [
+    ['lightStatus/xray',0.5,1.5,'filter1'],
+    ['ipm_dg1/sum',1e3,4.5e4,'filter1'],
+    # ['ipm3/sum',200,3e3,'filter1'],
+    # ['tt/FLTPOSFWHM',60,230,'filter1'],
+    # ['tt/AMPL',0.005,0.19,'filter1']
+    # ['evr/code_41',0.5,1.5,'custom']
+]
+
+# Laser on/off
+laser = False
+
+# ##### DETECTORS ########################################################################
+# List detectors to be cubed. Area detector have additional options such as threshold
+# Full images will always be saved. 
+# More area detector can be defined following the same syntax, and adding them to varList
+detDict = {'source':'Rayonix', 
+           'full':1, 
+           'image':1, 
+           'thresADU':-1e5, 
+           'common_mode':0}
+
+pix_size = 176e-6
+func_kwargs = {
+    'name': 'azav_pyfai', # must be the name of the smalldata_tools ana_func
+    'ai_kwargs': {'dist':1, 'poni1':960*pix_size, 'poni2':960*pix_size},
+    'npts': 512,
+    'int_units' : '2th_deg',
+    'return2d' : False
+}
+det_proc = [func_kwargs]
```

Comment:
Just to make sure, if I wanted to add two functions, I'd add a second dict? Do you want to add a comment line stating that (not that this would be often used...)? Does saving the data in image format now also follow this convention (I think this would be good for consistency).

Reply:
det_proc is a list of dictionaries, each defining a post-processing function. So you can have as many as you want.
The post-processing function is applied to the summed data in the bins, so if you pick the image option, it will be applied to the summed image, if not, it will be applied to the calib data.

Reply:
What I mean is: win my old code, how image worked is that I used code convert the summed calib-data to an image only at the very end, so really exactly how imageFunc works on event-by-event data. Now that you have a much better way of applying analysis functions on the summed data, maybe it's time to have the cube only work on calib and define making the image as a post processing function. When of course we need to make an onion-type arrangements that we could run an azimuthal averaging function on top of the image function which runs on the summed caliber data. 
It's fine how it is, I'm not actually sure what I suggested is better or just different?

Reply:
I'll resolve this now as this is something that could be done later if that is even the right choice.

Reply:
What you describe is actually how it works: it only applies the image function at the very end, if image=1 is chosen in the detector config.